cmd/editorconfig-checker/main.go

```diff
@@ -78,6 +89,17 @@ func parseArguments() {
 	cmdlineConfig = config.Config{}
 	writeConfigFile = false
 
+	// check the NO_COLOR environment variable before parsing the arguments, so the arguments can override
+	if nocolor := os.Getenv("NO_COLOR"); nocolor != "" {
+		if nocolorParsedAsBool, err := strconv.ParseBool(nocolor); err == nil {
+			if nocolorParsedAsBool {
+				enableNoColor("")
+			}
+		} else {
+			enableNoColor("")
+		}
+	}
```

Comment:
This would be clearer I think

```suggestion
	// check the NO_COLOR environment variable before parsing the arguments, so the arguments can override
	if nocolor := os.Getenv("NO_COLOR"); nocolor != "" {
		nocolorParsedAsBool, err := strconv.ParseBool(nocolor)
		if err != nil {
			// let's assume assume the user wanted the NO_COLOR
			nocolorParsedAsBool = true
		}
		if nocolorParsedAsBool {
			enableNoColor("")
		}
	}
```